src/expandable-section/styles.scss

```diff
@@ -102,13 +101,19 @@
   &-button {
     box-sizing: border-box;
     display: flex;
+  }
 
-    // Header buttons for non-container variants show the focus ring directly
+  &-button,
```

Comment:
Now the focus styles of container variant are the same as the other variants (the focus ring only wraps the triangle icon and the header text in all cases), so some custom code specific to the container variant further down can be removed. The mouse cursor behavior however is still a bit custom for container variant because it will be restricted to the new smaller target area when interactive elements are present.